packages/react-magma-dom/src/components/NativeSelect/NativeSelect.test.js

```diff
@@ -34,4 +36,56 @@ describe('NativeSelect', () => {
       return expect(result).toHaveNoViolations();
     });
   });
+
+  it('should render a disabled select', () => {
+    const testId = 'test-id';
+    const { getByTestId } = render(
+      <NativeSelect disabled testId={testId}></NativeSelect>
+    );
+    const nativeselect = getByTestId(testId);
+    expect(nativeselect).toHaveStyleRule(
+      'color',
+      transparentize(0.4, magma.colors.neutral500)
+    );
+  });
+
+  it('should render a disabled inverse select', () => {
+    const testId = 'test-id';
+    const { getByTestId } = render(
+      <NativeSelect disabled isInverse testId={testId}></NativeSelect>
+    );
+    const nativeselect = getByTestId(testId);
+    expect(nativeselect).toHaveStyleRule(
+      'color',
+      transparentize(0.6, magma.colors.neutral100)
+    );
+  });
+
+  it('should render an error state', () => {
+    const testId = 'test-id';
+    const { getByTestId } = render(
+      <NativeSelect
+        errorMessage="This is an error"
+        testId={testId}
+      ></NativeSelect>
+    );
+    const nativeselect = getByTestId(testId).parentElement;
+    expect(nativeselect).toHaveStyleRule('border-color', magma.colors.danger);
+  });
```

Comment:
This is a almost a duplicate of lines 18-23. They can be condensed to 1

Reply:
This is missing `expect(getByText(errorMessage)).toBeInTheDocument();` (and the inverse test below as well)